actions/websocket_actions.jsx

```diff
@@ -373,7 +373,50 @@ function handleChannelMemberUpdatedEvent(msg) {
     dispatch({type: ChannelTypes.RECEIVED_MY_CHANNEL_MEMBER, data: channelMember});
 }
 
-function handleNewPostEvent(msg) {
+export function debouncePostEvent(func, wait) {
+    let timeout;
+    let queue = [];
+    return function fx(...args) {
```

Comment:
Some more nitpicking, but since we're just calling this with a single websocket event each time, maybe avoid spreading the arguments here?
```
    return function fx(msg) {
```
If reselect is anything to go by, they have notes in the source for `createSelector` about using `arguments` instead of spreading for performance reasons, although since we know there's only one parameter, we don't even need to go that far

Reply:
Yup. It was originally meant to be generic. But that didn't work out.